src/Diagnostics/FxCop/Desktop.Analyzers/CSharp/Desktop.CSharp.Analyzers.props

```diff
@@ -0,0 +1,22 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- 
+    This file contains the rules that have been implemented in this package and therefore should be disabled for the binary FxCop.
+    When a new rule is implemented in this package, make sure to add an entry below to disable that rule in FxCop to avoid duplicate 
+    issues.
+    The format is -[Category]#[ID]
+  -->
+  <PropertyGroup>
+    <CodeAnalysisRuleSetOverrides>$(CodeAnalysisRuleSetOverrides);
+      -Microsoft.Design#CA1003;
+      -Microsoft.Design#CA1019;
+
+      -Microsoft.Globalization#CA1309;
+
+      -Microsoft.Performance#CA1820;
+
+      -Microsoft.Reliability#CA2002;
+
+      -Microsoft.Usage#CA2213;
+    </CodeAnalysisRuleSetOverrides>
```

Comment:
We really need a Roslyn analyzer to catch this. I think almost everyone ends up not updating this file correctly, let me open a post 1.0 issue.